src/main/java/Duke.java

```diff
@@ -15,9 +15,22 @@ public static void main(String[] args) {
         // reading user input
         Scanner sc = new Scanner(System.in);
 
-        // boolean to indicate whether userInput isBye
+        // INITIALIZING VARIABLES
+        /**
+         * whether the user's input is bye
+         */
         boolean isBye = false;
 
+        /**
+         * current number of to do tasks
+         */
+        int count = 0;
```

Comment:
Maybe instead of describing count variable in the javadoc comments you can name it something like "numOfTasks"?